pxf/pxf-hive/src/main/java/org/apache/hawq/pxf/plugins/hive/HiveMetadataFetcher.java

```diff
@@ -136,7 +136,7 @@ public HiveMetadataFetcher(InputData md) {
     private OutputFormat getOutputFormat(String inputFormat, boolean hasComplexTypes) throws Exception {
         OutputFormat outputFormat = null;
         InputFormat<?, ?> fformat = HiveDataFragmenter.makeInputFormat(inputFormat, jobConf);
-        String profile = ProfileFactory.get(fformat, hasComplexTypes);
+        String profile = ProfileFactory.get(fformat, hasComplexTypes, null);
```

Comment:
passing explicit null params should be avoided, if possible, override the function if more/less params are desired.